src/components/WordSearch/WordSearch.tsx

```diff
@@ -421,6 +422,7 @@ const WordSearch = ( wordsProp: WordSearchProps ) => {
 
   return (
     <div className={styles.gridContainer}>
+      {correctWords.length == wordList.length ?  <CompleteModal /> : '' }
```

Comment:
```
{correctWords.length === wordList.length &&
<CompleteModal />
}
```

can we make it like this instead, so that if false it didnt render empty string

Reply:
Will do, Thank you for giving suggestion.